pagespeed/envoy/utility.cc

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Do we really need to copy utility.cc/h?

Reply:
remove this file
code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/4ab87afdc150b403bcef176c8f4934e8f4ba18d1